site/src/components/page-faq/helpscout-webhook.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * function to submit form data to helpscout via zapier webhook
+ */
+export const submitToHelpScout = async ( data: Object)
+   : Promise<true | {
+      statusText: string;
+      status: number;
+      data?: any;
+   }> => {
+   // link to associated zap https://zapier.com/app/zap/100408573
+   const response = await fetch( "https" + "://hooks.zapier.com/hooks/catch/8459352/ow03eoe/", {
+      method: "POST",
+      body: JSON.stringify(data),
+      mode: "no-cors",
```

Comment:
If possible send a normal request so we can get the response status for the below.

Reply:
changed mode to cors.